### ISSUE TITLE
Auto alert

### ISSUE THREAD

**Issue by tjoozey** [fabien-d/alertify.js#110]
How do I create an external alertify.js using tools provided in the web.

I was looking to have an ok button, and the prompt reading as follows:

<b>You have reached the WW Network.<Br>The WW Network uses state-of-art tracking from our private domain tracker.ww.com.<br><i>We are only accepting ten new affiliates for December, so join today!</i>

Regards.

**Comment by fabien-d** [comment]
> How do I create an external alertify.js using tools provided in the web.

Not sure what you mean?

From what I get you should be able to just run `alertify.alert("You have...");`

**Comment by tjoozey** [comment]
Hello,

This is the code im using to create a simple alert that is prompted automatically when a visitor gives a page hit.

<head><meta charset="utf-8">
<link rel="stylesheet" href="http://fabien-d.github.com/alertify.js/assets/js/lib/alertify/alertify.core.css" />
<link rel="stylesheet" href="http://fabien-d.github.com/alertify.js/assets/js/lib/alertify/alertify.default.css" />

<script src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.8.3.min.js"></script>

<script src="http://fabien-d.github.com/alertify.js/assets/js/lib/alertify/alertify.min.js"></script>

<script> $(document).ready(function() { 
    // alert dialog
    alertify.alert("<b>You have reached the WW Network.<Br>The WW Network uses state-of-art<a href="http://www.hasoffers.com/customer-satisfaction/">tracking</a> from our private domain tracker.ww.com.<br><i>We are only accepting<a href="http://network.w-wnetwork.com/signup">*10* new affiliates for January, so join today!</i>");
</script>

Can you tell me how to get this dialog to work? Also, when User hits  the label "ok", how do I make it redirect to a URL?

Awaiting your reply.

**Comment by fabien-d** [comment]
In order to redirect to a different URL, you need to use the callback function, see below:

``` js
$(document).ready(function() {
    // alert dialog
    alertify.alert("<b>You have reached the WW Network.<Br>The WW Network uses state-of-art<a href=\"http://www.hasoffers.com/customer-satisfaction/\">tracking</a> from our private domain tracker.ww.com.<br><i>We are only accepting<a href=\"http://network.w-wnetwork.com/signup\">*10* new affiliates for January, so join today!</i>", function () {
        window.location = "http://google.ca";
    });
});
```

note: you have to escape `\"` quotes or use single quotes.

**Comment by tjoozey** [comment]
This {{URL}} needs to be opened in a new tab/window.

Does alertify.js have this function?

Regards  

## 

Tyler Johnson
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by fabien-d** [comment]
I pasted the exact code you need to run what you're asking for in the comment above. You just need to replace the `window.location` with `window.open(URL)`.

I'd rather you not post signup links in the comments for self promotion. I'm glad to help hooking up the functionality you need - but that doesn't require you posting the actual URL.

Cheers,
